tests/spatial/rotation_test.py

```diff
@@ -10,12 +11,13 @@
 
 def test_from_rotvec_bad_unit():
     with pytest.raises(sc.UnitError):
-        rotation_from_rotvec(value=[90, 0, 0], unit="m")
+        rotations_from_rotvecs(rotation_vectors=sc.vector(value=[90, 0, 0], unit="m"))
```

Comment:
I think when we switch to Python 3.8 we would probably make this single argument position-only. I'd not use a keyword arg here?